### PR TITLE
Update doc for analytics observation to fix property name

### PR DESCRIPTION
**Pull request by iujames**
Noticed while working on Ionic side that the doc here about the receiver side used `analyticType` but the actual implementation of the event emitter uses `analytic` for that property - [iOS](https://github.com/appcues/appcues-react-native-module/blob/main/ios/AppcuesReactNative.swift#L131) and [Android](https://github.com/appcues/appcues-react-native-module/blob/main/android/src/main/java/com/appcuesreactnative/AppcuesReactNativeModule.kt#L62).  I believe these have to match.

### CODE REVIEW

File: docs/AnalyticObserving.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ eventListener.remove();
 
 ## Using the Analytics Tracking Data
 
-There are four different possible values for `analyticType` and two possible values for `isInternal`:
+There are four different possible values for `analytic` and two possible values for `isInternal`:
 
-| `analyticType` | `isInternal` | SDK call              |
+| `analytic` | `isInternal` | SDK call              |
```
Comment (mmaatttt) [inline docs/AnalyticObserving.md:34]: Fix the table spacing, pretty please 😁

**Review by mmaatttt — APPROVED** [review]
Whoops, thanks. If `analyticType` is a clearer name, we can update to that too...

File: docs/AnalyticObserving.md (hunk 1 above)
Comment (iujames) [reply docs/AnalyticObserving.md]: woops, updated

**Comment by iujames** [comment]
> Whoops, thanks. If `analyticType` is a clearer name, we can update to that too...

no strong pref, I'm fine with `analytic`, was building that on Ionic side to match.

**Pull request merged**